kotoed-js/src/main/ts/profile/edit.tsx

```diff
@@ -200,6 +208,57 @@ export class ProfileComponent extends ComponentWithLocalErrors<ProfileComponentP
         </div>;
     };
 
+    renderDiffPreference = () => {
+        return <div className="form-group">
+            <label htmlFor="diff-mode-preference" className="col-sm-2 control-label">Preferred diff mode</label>
+            <div className="col-sm-10">
+                <div className="radio">
+                    <label htmlFor="input-diff-mode-course-base">
+                        <input
+                            id={`input-diff-mode-course-base`}
+                            type="radio"
+                            name="diff-mode"
+                            value="COURSE_BASE"
+                            checked={this.props.denizen.diffModePreference == "COURSE_BASE"}
+                            onChange={this.bindRadio("diffModePreference")}
+                            defaultChecked={this.state.denizen.emailNotifications}
```

Comment:
`emailNotifications`?)

Reply:
Also a typo, not sure what is this `defaultChecked`.